docs/getting-started-step-by-step/02a-managed-resources-aws.md

```diff
@@ -0,0 +1,144 @@
+# Using Managed Resources from AWS: Connect Crossplane to manage AWS resources.
+
+In this part, our goal is to configure Crossplane so we can launch and manage
+AWS resources directly from within Kubernetes. We will leverage the power of the
+Kubernetes machinery that continuously reconciles the desired state without any
+manual interaction from users.  
+
+![architecture-s3-bucket](architecture-s3-bucket.jpg)
+
+* A) Install the AWS Provider
+* B) Configure the AWS Provider
+* C) Create, Update and Delete an S3 bucket
+
+## Prerequisite
+
+A running Kubernetes cluster & Crossplane runtime installed. See
+[01-setup.md](Setup). The [AWS CLI](https://aws.amazon.com/cli/) installed.
+
+## A) Install the AWS Provider
+
+Cloud provider resources are bundled into a Crossplane packages called Provider.
+We will install the AWS version v0.19:
+
+### Run:
+```Bash:
+# Install the AWS provider with the Crossplane CLI:
+$ kubectl crossplane install provider crossplane/provider-aws:v0.19
+```
+
+This provider package installed a couple of AWS specific CRDS which we will be
+using.
+
+### Check:
+```Bash:
+# Show all installed Provider:
+# (currently only the AWS provider)
+# (check for Installed & Healthy)
+$ kubectl get providers
+
+# Describe the AWS provider:
+# (check for events and conditions)
+$ kubectl describe provider provider-aws
+
+# Show the CRDs the Provider has installed: 
+$ kubectl get crds | grep aws.crossplane.io
+```
+
+## B) Configure the AWS Provider
+
+The Provider package from the previous step just contains the resources and
+needs to be configured. In this step we will install a ProviderConfig with a
+secret containing the AWS credentials:
+
+### Run:
+```bash
+# Create a creds.conf file with the aws cli:
+AWS_PROFILE=default && echo -e "[default]\naws_access_key_id = $(aws configure get aws_access_key_id --profile $AWS_PROFILE)\naws_secret_access_key = $(aws configure get aws_secret_access_key --profile $AWS_PROFILE)" > creds.conf
+
+# Create a secret with the AWS creds:
+$ kubectl create secret generic aws-creds -n crossplane-system --from-file=creds=./creds.conf
+
+# Configure AWS Provider by creating a ProviderConfig:
+$ kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/release-1.2/docs/snippets/configure/aws/providerconfig.yaml
+```
+
+### Check:
+```bash
+# Check secret: 
+$ kubectl get secret aws-creds -n crossplane-system -o yaml
+# 
+$ kubectl describe providerconfig
+```
+
+## C) Create, Update and Delete an S3 bucket
+
+In this last step we will be testing our AWS Provider by creating a S3 bucket,
+update it and delete it. 
+
+### Run:
+```bash
+# Create a (hopefully) unique S3 bucket name
+BUCKET_NAME=test-bucket-$RANDOM
+
+# Install an S3 bucket
+$ cat <<EOF | kubectl apply -f -
+apiVersion: s3.aws.crossplane.io/v1beta1
+kind: Bucket
+metadata:
+  name: $BUCKET_NAME
+spec:
+  forProvider:
+    acl: public-read-write
+    locationConstraint: us-east-1
+EOF
+```
```

Comment:
The established convention in our documentation is to avoid here documents wrapped in shell and instead present plain YAML, immediately followed by a pastable `kubectl apply` command. This lets the viewer see syntax highlighted YAML without arcane bash decoration, while still making it easy to paste a command. This is another pattern we landed on after much back and forth (we actually copied it from the Kubernetes documentation), so I'd prefer not to revert to using here docs unless there is a strong case to do so.

Examples: 
* https://github.com/crossplane/crossplane/blob/master/docs/getting-started/provision-infrastructure.md#claim-your-infrastructure
* https://kubernetes.io/docs/tutorials/stateless-application/guestbook/#start-up-the-redis-database

Reply:
Ack. The reason I've used this style is that I can verify the contents while applying them in the same step. But nothing to debate on in this PR. I've added  "Move resources into dedicated files and show their content separate from the kubectl apply. As per comment." as a style guide task.